solr/solrj/src/test/org/apache/solr/client/solrj/response/TestSpellCheckResponse.java

```diff
@@ -34,17 +38,26 @@
  * @since solr 1.3
  */
 public class TestSpellCheckResponse extends EmbeddedSolrServerTestBase {
+  private static SolrClient client;
```

Comment:
Do we need a static client?

Reply:
Maybe this is partly to avoid changing so many lines of the test?

Reply:
Indeed; some subclasses assumed this and we are restraining ourselves a bit from doing more work than needed.